presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures when ``ROW`` and ``ARRAY`` with ``NULL`` elements fail to compare.
+* Lazily build the hash table for map.
+
+Raptor Changes
+--------------
+* Integrate the optimized ORC writer into Raptor. Add configuration property ``storage.orc.optimized-writer-stage`` to enable and configure the optimized writer.
+  To switch to the optimized ORC writer is only backward compatible but not forward compatible.
+  Turning on the optimized ORC writer should work on an existing Raptor server. However, switching it back to Hadoop ORC writer can fail Raptor background job.
+* Integrate ``ZSTD`` compression for file writer. Add configuration property ``storage.orc.compression-kind`` to switch between ``SNAPPY`` and ``ZSTD``.
```

Comment:
```
Add support for writing files with ``ZSTD`` compression.  Add configuration ...
```
I'm assuming that "Integerate zstd compression for file writer" means adding support for writing zstd files, but I'm not actually certain.